### DISCUSSION TITLE
[Feature] Metrics - prefix with "immich_" to improve co-existence with other applications in Prometheus

### DISCUSSION THREAD

**Discussion by matthiasschoger** [immich-app/immich#13023] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, metrics keys use generic names which make sense in a stand-alone environment, but can become very confusing in a central Prometheus instance where a lot of applications metrics are stored.

My proposal would be to prefix all the Immich metrics with "immich_" to make it clear which metrics belong to Immich. 

I understand that this would be a breaking change. On the other hand, I don't think that the rather new metrics feature is not heavily used right now. Any delay in making that change would make it even harder in the future to switch to a more compatible metrics naming scheme.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8191
